Doc/Orbiter Developer Manual/PUBLISH.tex

```diff
@@ -0,0 +1,44 @@
+\documentclass[Orbiter Developer Manual.tex]{subfiles} 
+\begin{document}
+
+\section{Publishing addons}
+Now that you have created and tested your new ship, you want to share it with the rest of the Orbiter community. Here is how to do it:
+
```

Comment:
Maybe we should add some words about licencing in this section ?

Reply:
Makes sense. Added a "TODO" in that section to indicate the need for this (this PR is for LaTeX conversion work, the next PR will work the updating and checking of the content).